glean-core/ios/GleanTests/Scheduler/MetricsPingSchedulerTests.swift

```diff
@@ -317,11 +337,15 @@ class MetricsPingSchedulerTests: XCTestCase {
     }
 
     func testTimerInvocation() {
-        // Set up the expectation
-        expectation = expectation(description: "Timer fired")
-
         // Reset Glean clearing the stores so that the metrics ping will be empty
+        expectation = setUpDummyStubAndExpectation(testCase: self, tag: "MetricsPingSchedulerTests")
         Glean.shared.resetGlean(clearStores: true)
+        waitForExpectations(timeout: 5.0) { error in
```

Comment:
Same question again: what ping is this? The comment says we clear storage, so the metrics ping should be empty and not be sent.

Reply:
Just capturing the baseline->reason=active ping to prevent it from being transmitted to the ingestion pipeline.